### PR TITLE
feat: show help text

### PR DESCRIPTION
**Pull request by dbolson**
Show help text for every step. We want to support going back for each step before merging this.

Also, we should try to fix the padding in the list of SDKs separate from this change.

Screenshot 2024-04-05 at 10 06 55 AM
Screenshot 2024-04-05 at 10 07 00 AM
Screenshot 2024-04-05 at 10 07 09 AM
Screenshot 2024-04-05 at 10 07 15 AM

### CODE REVIEW

File: internal/quickstart/create_flag.go [hunk 1]
```diff
@@ -29,7 +33,18 @@ func NewCreateFlagModel(client flags.Client, accessToken, baseUri string) tea.Mo
 		accessToken: accessToken,
 		baseUri:     baseUri,
 		client:      client,
-		textInput:   ti,
+		help:        help.New(),
+		helpKeys: keyMap{
+			Back: key.NewBinding(
+				key.WithKeys("esc"),
+				key.WithHelp("esc", "back"),
+			),
```
Comment (k3llymariee) [inline internal/quickstart/create_flag.go:41]: we can't go back on the input step here though - with the new success page we will be able to but is there a way to hide it when we should the textinput View?

File: internal/quickstart/toggle_flag.go [hunk 2]
```diff
@@ -75,5 +92,5 @@ func (m toggleFlagModel) View() string {
 		Padding(0, 1).
 		MarginRight(margin)
 
-	return title + "\n\n" + toggleStyle.Render(toggle) + m.flagKey + furtherInstructions
+	return title + "\n\n" + toggleStyle.Render(toggle) + m.flagKey + furtherInstructions + "\n\n" + helpView
```
Comment (k3llymariee) [inline internal/quickstart/toggle_flag.go:95]: the help text jumps after the the flag is toggled for the first time. I don't think there's an easy way to pad this to make sure that doesn't happen though 😞

**Review by k3llymariee — APPROVED** [review]

File: internal/quickstart/toggle_flag.go (hunk 2 above)
Comment (dbolson) [reply internal/quickstart/toggle_flag.go:89]: I was able to add some padding to keep the help text on the same line, but I'm not sure if the additional space is better than the jump. Maybe we leave the jump to see if anybody comments on it?
Screenshot 2024-04-05 at 1 56 24 PM

Comment (k3llymariee) [reply internal/quickstart/toggle_flag.go:89]: sgtm!

File: internal/quickstart/create_flag.go (hunk 1 above)
Comment (dbolson) [reply internal/quickstart/create_flag.go]: I removed "back" from the help text for create flag.

**Pull request merged**